### PR TITLE
fix(main/{lit,luvit}): don't embed luvi into {lit,luvit} bin

### PR DESCRIPTION
**Pull request by mbekkomo**
This PR changes the way how `lit` package and `luvit` package are built and installed.

### CODE REVIEW

File: packages/luvit/build.sh [hunk 1]
```diff
@@ -3,10 +3,11 @@ TERMUX_PKG_DESCRIPTION="Asynchronous I/O for Lua"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=2.18.1
+TERMUX_PKG_REVISION=2
```
Comment (twaik) [inline packages/luvit/build.sh:6]: You jumped from "0" to "2" :)

Comment (mbekkomo) [reply packages/luvit/build.sh:6]: I thought every packages starts at `1`, or am I mistaken?

Comment (twaik) [reply packages/luvit/build.sh:6]: No, it starts without revision, and we add only if we update package with same version.

**Review by TomJo2000 — APPROVED** [review]
Should be good to merge now.

**Comment by Biswa96** [comment]
CI failed in merged commit.

**Comment by twaik** [comment]
I started it over. Now it is fine.

**Pull request merged**